src/components/OnyxProvider.js

```diff
@@ -12,6 +12,7 @@ const [withCurrentDate, CurrentDateProvider] = createOnyxContext(ONYXKEYS.CURREN
 const [withReportActionsDrafts, ReportActionsDraftsProvider] = createOnyxContext(ONYXKEYS.COLLECTION.REPORT_ACTIONS_DRAFTS);
 const [withBlockedFromConcierge, BlockedFromConciergeProvider] = createOnyxContext(ONYXKEYS.NVP_BLOCKED_FROM_CONCIERGE);
 const [withBetas, BetasProvider] = createOnyxContext(ONYXKEYS.BETAS);
+const [withReportCommentDrafts, ReportCommentDraftsProvider] = createOnyxContext(ONYXKEYS.COLLECTION.REPORT_DRAFT_COMMENT);
```

Comment:
@mkarkachov Is creating a new context for the report comment drafts key necessary?

Reply:
It's related to https://github.com/Expensify/App/pull/20730#discussion_r1233410220

Reply:
@mollfpr I prefer to use `withOnyx` version but I did not find how to avoid unit test errors